src/Export/Base/DispositionInlineTrait.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/*
+ * This file is part of the Kimai time-tracking app.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace App\Export\Base;
+
+use Symfony\Component\HttpFoundation\ResponseHeaderBag;
+
+trait DispositionInlineTrait
+{
+    /**
+     * @var string
+     */
+    private $disposition = ResponseHeaderBag::DISPOSITION_ATTACHMENT;
+
+    /**
+     * @return string
+     */
+    public function getDisposition(): string
+    {
+        return $this->disposition;
+    }
+
+    /**
+     * @param bool $useInlineDisposition
+     * @return void
+     */
```

Comment:
all redundant comments can be removed (redundant as in: already 100% clear through function definition)